packages/block-library/src/index.js

```diff
@@ -282,7 +282,7 @@ export const registerCoreBlocks = (
 	blocks.forEach( ( { init } ) => init() );
 
 	setDefaultBlockName( paragraph.name );
-	if ( window.wp && window.wp.oldEditor ) {
+	if ( window.wp && window.wp.oldEditor && blocks.includes( classic ) ) {
```

Comment:
I would just make it more foolproof by searching just for the name, like `.some( block => block.name === classic.name )`. Don't rely on the fact that `blocks` contains the exact `classic` object.

Reply:
Good idea 👍 Done in ec67e057b8621bdb37d61d669287d17753b77629